sql/mob_pools.sql

```diff
@@ -4737,7 +4737,7 @@ INSERT INTO `mob_pools` VALUES (4676,'Eald_narche_2','Eald_narche_2',394,0x00000
 INSERT INTO `mob_pools` VALUES (4677,'Achuka','Achuka',461,0x0000D30900000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,1,131,0,0,0,1,0,461);
 INSERT INTO `mob_pools` VALUES (4678,'Hurkan','Hurkan',460,0x0000140A00000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,1,129,0,0,0,1,0,460);
 INSERT INTO `mob_pools` VALUES (4679,'Ocythoe','Ocythoe',471,0x0000130800000000000000000000000000000000,4,4,6,240,100,0,1,0,0,2,0,0,7,1695,0,0,2,1,0,471);
-INSERT INTO `mob_pools` VALUES (4680,'Sugaar','Sugaar',192,0x0000DF0700000000000000000000000000000000,1,1,7,240,100,0,1,0,0,0,7,0,7,155,0,0,0,1,0,192);
+INSERT INTO `mob_pools` VALUES (4680,'Sugaar','Sugaar',192,0x0000DF0700000000000000000000000000000000,1,1,7,240,100,0,1,0,0,2,7,0,7,155,0,0,0,1,0,192);
```

Comment:
I think this NM also needs a non generic skill list, looks like it only uses [Grim Glower](http://ffxiclopedia.wikia.com/wiki/Grim_Glower) (I don't think we have that working yet) and [Torpefying Charge.](http://ffxiclopedia.wikia.com/wiki/Torpefying_Charge)

If I'm wrong there let me know - I've never met this NM on retail, so I only have [sh\*\*WikiSays](http://ffxiclopedia.wikia.com/wiki/Sugaar) to go on

Reply:
I never encountered it on retail either so I am unsure, but my thought process here was the person on the wiki was explicitly noting those because they are the "big threats" it has and it has normal moves too, but they just didn't note them.

Reply:
usually they'll says more like "uses standard peiste moves" if it has the full moveset and lists specific ones if those are all it has. most pages will just list them all even if that amounts to all the moves that mob family has anyway. NMs often have fewer movebs but the ones they have often hit harder.

We can get someone to check retail if you use discord https://discord.gg/sX9JgDT

Reply:
Thanks, request posted on discord and once we get that information I'll do my best to adjust accordingly.

Reply:
From the wiki page here it looks to use all tp moves but Oppresive Gaze. 
The tp move Grim Glower is a sunanimation gaze attack that while in effect if you are face to face you become petrified the entire time to sub is active. I dont believe this can be coded yet. 
http://ffxiclopedia.wikia.com/wiki/Category:Peistes

Reply:
thats the family page, it doesn't say anything specific to this one NM other than its name where it lives and what it drops.

Reply:
this should be able to be coded... why not? we have scripts that can check to see if you're facing mob for other "gaze type" moves... it's not like the petrification just "comes off" once it stops looking at you... you get petrified for a set time ~15sec and then once it wears you get it back if you're still looking at it's glowing eyes... should be doable

Reply:
prolly onmobfight i'd guess
if mob has subanimation GloweyEyesThingy and mob isfacing target and mob target isfacing mob then give petrification... is where i would start

Reply:
No not onMobFight we'd do the same thing we need to support aura's except check both entities are facing and within a cone. So a large part of this happens in core which is why ppl haven't done it yet.

Reply:
Its honestly not real hard but ppl who know how haven't had both time and interest at the same time yet

Reply:
https://youtu.be/dfCXNfkOObo
I did this with dsp master just the way it is... no core changes... only lua scripting in onMobFight